tests/test_trial.py

```diff
@@ -25,6 +25,25 @@
     [storages.InMemoryStorage, lambda: storages.RDBStorage('sqlite:///:memory:')])
 
 
+@parametrize_storage
+def test_check_distribution_suggest_float(storage_init_func):
```

Comment:
I added the test to check if:

- `suggest_float(...)` and `suggest_uniform(...)` are the same
- `suggest_float(..., log=True)` and `suggest_loguniform(...)` are the same

Reply:
I think checking the compatibility with the existing interface is enough for the first implementation. 

And also, I think we need to check the case `low==high`.

Reply:
34d60b1
I added the test for checking the case `low==high`.